releasenotes/notes/aiohttp-98ae9ce70dda1dbc.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    aiohttp: add client integration. This integration traces requests made using the aiohttp client and includes support
+    for distributed tracing. See :ref:`the documentation<aiohttp>` for more information.
```

After dealing with broken references in release notes, I would be inclined to avoid them here.

```suggestion
    for distributed tracing.
```

---

ah darn 😕